docs-src/community.scrbl

```diff
@@ -0,0 +1,35 @@
+#lang scribble/manual
+@(require "lib.rkt")
+@(mint-scope "Community")
+
+@title[#:version reach-vers #:tag "community"]{Community}
+
+Below are links to Reach-related instructional content from the Reach community. To submit a link for consideration, contact mhagen@"@"reach.sh.
```

Comment:
We use semantic newlines (i.e. add a newline after `.;,` and only then)

Reply:
Please change to active voice "If you would like to..." and don't have your email, instead say to submit a Github issue and we'll add it.